WooCommerce/Classes/System/SessionManager.swift

```diff
@@ -185,8 +185,11 @@ final class SessionManager: SessionManagerProtocol {
 
         defaultStoreIDSubject = .init(defaults[.defaultStoreID])
 
-        // Listens when the core data stack is rest.
-        NotificationCenter.default.addObserver(self, selector: #selector(handleStorageDidReset), name: .StorageManagerDidResetStorage, object: nil)
```

Comment:
❓  Maybe this observation of `StorageManagerDidResetStorage` can still be kept, since this notification is still posted after destroying the database in `createPersistentContainer`'s recovery upon error. Or do we also want to replace `StorageManagerDidResetStorage` with the new notification there for consistency? I'm not sure if `StorageManagerDidResetStorage` notification has always been posted in `createPersistentContainer` from git as there was some refactoring on the file.

https://github.com/woocommerce/woocommerce-ios/blob/68f756ae01b4496c0541c9b0833b6abac48c465b/Storage/Storage/CoreData/CoreDataManager.swift#L187-L190

Reply:
Good point! This has been restored in 7109840a050fc2636ad189518f22421a620cd614.